### PR TITLE
changed get_startup_script to run also on SSH

### PR DESCRIPTION
**Pull request by fgelcer**
* added a command to increase the ulimit to 500k open files
* moved the MaxSessions setting to all transports

it is a minor change, and i'm testing it right now.

### CODE REVIEW

**Comment by fgelcer** [comment]
@fruch , shall we backport to 3.1 as well?

**Comment by fgelcer** [comment]
@roydahan , updated the command to update the content of the /etc/security/limits.d/20-nproc.conf instead of /etc/security/limits.conf.

File: sdcm/cluster.py [hunk 1]
```diff
@@ -272,9 +272,16 @@ def get_startup_script(cls):
...
             post_boot_script += dedent('''
-                   sudo echo 'action(type="omfwd" Target="{0}" Port="{1}" Protocol="tcp")'>> /etc/rsyslog.conf
-                   sudo systemctl restart rsyslog
-                   '''.format(*cls.RSYSLOG_ADDRESS))
+                sudo echo 'action(type="omfwd" Target="{0}" Port="{1}" Protocol="tcp")'>> /etc/rsyslog.conf
+                sudo systemctl restart rsyslog
+            '''.format(*cls.RSYSLOG_ADDRESS))
+        post_boot_script += dedent('''
+               sudo sed -i 's/#MaxSessions \(.*\)$/MaxSessions 1000/' /etc/ssh/sshd_config
+               sudo systemctl restart sshd
```
Comment (bentsi) [inline sdcm/cluster.py]: did you tests this? I don't how sshd would react to restart during execution of the script using itself :)

Comment (fgelcer) [reply sdcm/cluster.py]: Yes, tested it, and the thing here is that this script is being executed in the instance initiation.
In any case, you can restart sshd service, as it won't allow new connections (while restarting) but it won't disconnect you from an existing one.

**Comment by bentsi** [comment]
@fgelcer  please resolve conflicts

**Comment by fgelcer** [comment]
> @fgelcer please resolve conflicts

@bentsi , done

**Comment by roydahan** [comment]
@bentsi please do not merge till I approve

**Comment by bentsi** [comment]
@roydahan please review

**Comment by fgelcer** [comment]
@roydahan ping

**Pull request merged**